nccs/pipeline/direct/agriculture.py

```diff
@@ -16,6 +16,48 @@
 IrrigationType = typing.Literal["firr", "noirr"]
 
 
+class ImpfRelativeCropyieldNCCS(ImpfRelativeCropyield):
+    """Impact functions for agricultural droughts."""
+    @classmethod
+    def impf_relativeyield(cls):
+        """Impact functions defining the impact as intensity
+
+        Returns
+        -------
+        impf : climada.entity.impact_funcs.ImpfRelativeCropyield instance
+        """
+        impf = cls()
+        impf.haz_type = 'RC'
+        impf.id = 1
+        impf.name = 'Relative Cropyield ISIMIP'
+        impf.intensity_unit = ''
+        # intensity = 0 when the crop production is equivalent to the historical mean
+        # intensity = -1 for a complete crop failure
+        # intensity = 1 for a crop production surplus of 100%
+
+        # the impact function covers the common stretch of the hazard intensity
+        # CLIMADA interpolates linearly in case of larger intensity values
+        impf.intensity = -1 * np.arange(-1, 10)
+        impf.mdr = (impf.intensity)
+        impf.mdd = (impf.intensity)
+        impf.paa = np.ones(len(impf.intensity))
+
+        return impf
```

Comment:
```suggestion
```
My understanding is that this class isn't used anywhere, is that right? Or is it part of an analysis notebook I can't see?